nestkernel/node_manager.cpp

```diff
@@ -851,6 +851,38 @@ NodeManager::prepare_nodes()
   LOG( M_INFO, "NodeManager::prepare_nodes", os.str() );
 }
 
+void
+NodeManager::post_run_cleanup()
+{
+#ifdef _OPENMP
+#pragma omp parallel
+  {
+    index t = kernel().vp_manager.get_thread_id();
```

Comment:
1) Yes, it could be. However, I just copied the code from the `finalize()` function and did not want to change or refactor the functionality at this point. The code will also go away when nestio is merged.